Source/help.cpp

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file help.cpp
+ *
+ * Implementation of the ingame help text.

```

Comment:
```suggestion
 * Implementation of the in-game help text.
```

Apply same fix to other places too.